server/spec/candlepin_scenarios.rb

```diff
@@ -502,3 +506,23 @@ def create_candlepin_export_update_no_ent
     create_candlepin_export()
   end
 end
+
+class AsyncStandardExporter < StandardExporter
+
+  def initialize
```

Comment:
You can drop this `initialize` if you want.  Since it just calls `super` it's not really doing anything different from the default Ruby behavior.